src/MembershipsSDKAdapter.js

```diff
@@ -24,6 +24,7 @@ function getMembers(members) {
       inMeeting: member.isInMeeting,
       muted: member.isAudioMuted,
       sharing: member.isContentSharing,
+      orgID: member.orgID,
```

Comment:
move this up just after `id` please, so we keep the same order as in the interface